kubernetes/namespaces/monitoring/alerts/alerts.d/etcd.yaml

```diff
@@ -0,0 +1,11 @@
+groups:
+  - name: etcd
+    rules:
+      - alert: EtcdErrorsSpike
+        expr: rate(etcd_request_error_total[5m]) > 0
+        for: 5m
+        labels:
+          severity: page
+        annotations:
+          summary: "Etcd request errors are spiking"
+          description: "Etcd request errors are spiking on {{ $labels.instance }} for {{ $labels.operation }}s to {{ $labels.type }}"
```

Comment:
Can we do anything about these? It feels like a Kubernetes-internal thing that will drive one of us crazy at 3 AM when LKE does their production rollout.

Reply:
This should not change as a result of anything LKE do, if we experience this it should be something that we can fix by recycling a node and it is often a precursor to wider network issues on a node.